forge-gui/res/cardsfolder/upcoming/yenna_redtooth_regent.txt

```diff
@@ -2,10 +2,9 @@ Name:Yenna, Redtooth Regent
 ManaCost:2 G W
 Types:Legendary Creature Elf Noble
 PT:4/4
-A:AB$ Pump | Cost$ 2 T | ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield | TgtPrompt$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SorcerySpeed$ True | SubAbility$ DBCopy | SpellDescription$ Choose target enchantment you control that doesn't have the same name as another permanent you control.
-SVar:DBCopy:DB$ CopyPermanent | Defined$ Targeted | RememberTokens$ True | NonLegendary$ True | SubAbility$ DBUntap | StackDescription$ Create a token that's a copy of it, except it isn't legendary. If the token is an Aura, untap CARDNAME, then scry 2. Activate only as a sorcery.
+A:AB$ CopyPermanent | Cost$ 2 T | Defined$ Targeted | RememberTokens$ True | NonLegendary$ True | ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield | TgtPrompt$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SorcerySpeed$ True | SpellDescription$ Choose target enchantment you control that doesn't have the same name as another permanent you control. | SubAbility$ DBUntap
```

Comment:
Change Defined to ValidTgts$ Enchantment.YouCtrl+doesNotShareNameWith OtherYourBattlefield

Reply:
But I tried this both of these ways and couldn't get it to work

Reply:
Did you play test this?